pkgs/applications/version-management/git-and-tools/git/default.nix

```diff
@@ -70,7 +72,9 @@ stdenv.mkDerivation {
   # so that `SPARSE_FLAGS' corresponds to the current architecture...
   #doCheck = true;
 
-  installFlags = "NO_INSTALL_HARDLINKS=1";
+  installFlags = "NO_INSTALL_HARDLINKS=1"
+    + (if withpcre2 then " USE_LIBPCRE2=1" else "");
```

Comment:
Do you know, where git uses libpcre2? I assume it replaces a slower posix regex engine somewhere.